### PR TITLE
tree-wide: fix typo

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -282,7 +282,7 @@ CHANGES WITH 249 in spe:
         * A new ConditionOSRelease= setting has been added to unit files to
           check os-release(5) fields. The "=", "!=", "<", "<=", ">=", ">"
           operators may be used to check if some field has some specific value
-          or do a aphanumerical comparison. Equality comparisons are useful for
+          or do the alphanumerical comparison. Equality comparisons are useful for
```
Comment (keszybz) [inline NEWS]: "a" is correct here, "the" is out of place.

File: man/systemd.link.xml [hunk 2]
```diff
@@ -144,7 +144,7 @@
       <varlistentry id='property'>
         <term><varname>Property=</varname></term>
         <listitem>
-          <para>A whitespace-separated list of udev property name with its value after a equal
+          <para>A whitespace-separated list of udev property name with its value after an equal
           (<literal>=</literal>). If multiple properties are specified, the test results are ANDed.
```
Comment (keszybz) [inline man/systemd.link.xml:148]: Preexisting, but "equal" → "equals sign".

File: man/systemd.socket.xml [hunk 3]
```diff
@@ -99,8 +99,8 @@
     <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>), receiving only
     the sockets configured through socket-activation from the host's namespace. In such a set-up communication within
     the host's network namespace is only permitted through the activation sockets passed in while all sockets allocated
-    from the service code itself will be associated with the service's own namespace, and thus possibly subject to a a
-    much more restrictive configuration.</para>
+    from the service code itself will be associated with the service's own namespace, and thus possibly subject to
+    much more restrictive configurations.</para>
```
Comment (keszybz) [inline man/systemd.socket.xml]: "a restrictive configuration" was better here.

File: man/systemd.link.xml (hunk 2 above)
Comment (mbiebl) [reply man/systemd.link.xml:148]: shouldn't that be plural? "... list of property names with their ...."

File: docs/USER_RECORD.md [hunk 4]
```diff
@@ -689,7 +689,7 @@ The following fields are defined in this section:
 hex. If any of the specified IDs match the system's local machine ID
 (i.e. matches `/etc/machine-id`) the fields in this object are honored.
 
-`matchHostname` → An array of string with a each a valid hostname. If any of
+`matchHostname` → An array of string with each a valid hostname. If any of
```
Comment (mbiebl) [inline docs/USER_RECORD.md]: This sentence reads a bit weird, I'm actually not sure what it's supposed to mean.

File: NEWS (hunk 1 above)
Comment (yuwata) [reply NEWS]: so, "an"? instead of "a"?

Comment (keszybz) [reply NEWS]: "an". Sorry!

File: docs/USER_RECORD.md (hunk 4 above)
Comment (keszybz) [reply docs/USER_RECORD.md]: "An array of strings, each one a valid hostname." ?

Comment (keszybz) [reply docs/USER_RECORD.md]: Or maybe "An array of strings that are valid hostnames.".

**Comment by yuwata** [comment]
@keszybz and @mbiebl Thank you for your review. I hope all above points are addressed. But please re-review carefully. My English is always bad...

**Pull request merged**